python/ray/serve/controller.py

```diff
@@ -165,6 +165,18 @@ def _stop_one_running_replica_for_testing(self, deployment_name):
             deployment_name
         ]._stop_one_running_replica_for_testing()
 
+    def _get_slow_startup_warning_period_s(self) -> float:
+        return ray.serve.deployment_state.SLOW_STARTUP_WARNING_PERIOD_S
+
+    def _get_slow_startup_warning_s(self) -> float:
+        return ray.serve.deployment_state.SLOW_STARTUP_WARNING_S
+
+    def _set_slow_startup_warning_period_s(self, period: float) -> None:
+        ray.serve.deployment_state.SLOW_STARTUP_WARNING_PERIOD_S = period
+
+    def _set_slow_startup_warning_s(self, time_limit: float) -> None:
+        ray.serve.deployment_state.SLOW_STARTUP_WARNING_S = time_limit
```

Comment:
- looks like they are called in together. Can you group them into the one _get and one _set call?
- do we need to configure them this way? How about support configure these via environment variable and and test with environment fixture (@shrekris-anyscale can help here)

Reply:
additionally, for private apis, we want to extremely pedantic, in particular, adding a `_for_testing` suffix.